src/terrain/routes/schemas/dashboard_aggregator.clj

```diff
@@ -4,60 +4,142 @@
+
+   (optional-key :status)
+   (describe (maybe String) "The current status of the analysis")
+
+   (optional-key :subdomain)
+   (describe (maybe String) "The subdomain assigned to the analysis. VICE only")
+
+   (optional-key :parent_id)
+   (describe (maybe UUID) "The UUID of the parent analysis. Only for batch analyses")})
 
 (defschema DashboardFeedItem
-  {:id                     (describe String "The unique identifier for a feed. Probably not a UUID")
-   :name                   (describe String "The name of the item in the feed. Not the name of the feed itself")
-   :description            (describe String "Corresponds to the content snippet provided by the feed. Named description to match the other types")
-   :link                   (describe String "Link to the source of the feed item. Probably leads to the website")
-   :date_added             (describe String "The date the item was added to the feed")
-   :publication_date       (describe String "The date the item was originally published")
-   :author                 (describe String "The author of the item")
-   (optional-key :content) (describe (maybe String) "The content of the item")})
+  {:id
+   (describe String "The unique identifier for a feed. Probably not a UUID")
+
+   :name
+   (describe String "The name of the item in the feed. Not the name of the feed itself")
+
+   :description
+   (describe String (str "Corresponds to the content snippet provided by the feed. Named description to match "
+                         "the other types"))
+
+   :link
+   (describe String "Link to the source of the feed item. Probably leads to the website")
+
+   :date_added
+   (describe String "The date the item was added to the feed")
+
+   :publication_date
+   (describe String "The date the item was originally published")
+
+   :author
+   (describe String "The author of the item")
+
+   (optional-key :content)
+   (describe (maybe String) "The content of the item")})
 
 (defschema DashboardFeeds
-  {(optional-key :news)   (describe (maybe [DashboardFeedItem]) "The news feed")
-   (optional-key :events) (describe (maybe [DashboardFeedItem]) "The events feed")
-   (optional-key :videos) (describe (maybe [DashboardFeedItem]) "The videos feed")})
+  {(optional-key :news)
+   (describe (maybe [DashboardFeedItem]) "The news feed")
+
+   (optional-key :events)
+   (describe (maybe [DashboardFeedItem]) "The events feed")
+
+   (optional-key :videos)
+   (describe (maybe [DashboardFeedItem]) "The videos feed")})
 
 (defschema DashboardAggregatedApps
-  {(optional-key :recentlyAdded) (describe (maybe [DashboardApp]) "Apps recently added by the user")
-   :public                       (describe [DashboardApp] "Apps recently made public")})
+  {(optional-key :recentlyAdded)
+   (describe (maybe [DashboardApp]) "Apps recently added by the user")
+
+   (optional-key :recentlyRan)
+   (describe (maybe [DashboardApp]) "Public apps that were recently used in jobs")
```

Comment:
This is the only substantive change in this PR.

Reply:
I probably missed it in the other PR since I'm not super familiar with app queries, but will this only return recently run public apps and not private apps as well?  This might go back to your comment in the other PR about whether this feature is supposed to show trending apps or recently used apps (which I'm not 100% sure, but thought it was for recently used apps).

Reply:
The feature I requested was recently used apps by user (public or private). Trending apps will need to come from the `analytics`

Reply:
Yeah, I implemented the wrong feature. 😖